packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx

```diff
@@ -12,8 +12,8 @@ export const useAppShortcuts = () => {
 
         // press CMD + P to show PassphraseModal
         if (modKey && e.key === 'p' && device) {
-            dispatch(createDeviceInstance({ device }));
-            e.preventDefault(); // prevent default behaviour
+            dispatch(createDeviceInstance({ device, useEmptyPassphrase: false }));
+            e.preventDefault(); // prevent default behavior
```

Comment:
Actually the previous one was correct

Reply:
![image](https://github.com/trezor/trezor-suite/assets/152899911/48d4f487-7cee-45ab-877a-0b7d349ffbb6)

This is what spell-checked in VSCode wants :D

:grin: